src/frontend/src/lib/components/hero/Header.svelte

```diff
@@ -9,10 +9,10 @@
 	import OisyWalletLogo from '$lib/components/icons/OisyWalletLogo.svelte';
 	import { authSignedIn } from '$lib/derived/auth.derived';
 	import { modalAboutHow, modalAboutWhat } from '$lib/derived/modal.derived';
-	import { isSubRoute } from '$lib/utils/nav.utils';
+	import { isRouteSettings } from '$lib/utils/nav.utils';
 
 	let back = false;
-	$: back = isSubRoute($page);
+	$: back = isRouteSettings($page);
```

Comment:
Not sure about testing solely for `settings` within this `Header` component, feels like something we might forget if we ever add more pages. Would it be worth having a prop?

Also `transactions` page has no back button anymore?

Reply:
- yes, makes sense! add a back `prop` that is given directly in `+layout`?

- the `transactions` page will have it automatically because it is included in `summary` that is defined as `summary={route === 'transactions'}` on `+layout`. My idea is to change this after: make it so summary is not passed down, but it is calculated directly in the component as `summary=isTransactionPage(`... do you want me to do it in a PR before this one (it will be in `main`)?

Reply:
>  add a back prop that is given directly in +layout?

yeah for example

> My idea is to change this afte

Coolio

> do you want me to do it in a PR before

Another PR sounds good

Reply:
@peterpeterparker have a look at the last commit please

Reply:
Did you push?

Reply:
I think so yes, should be this commit

https://github.com/dfinity/oisy-wallet/pull/2401/commits/0610a1b471074d40ca74fffc818605dfd97e0817

Reply:
I see it now, thx, but, shouldn't we use it here as well?

```
let back = false;
$: back = isRouteSettings($page);
```

Reply:
I refactored it a bit, have a look please